swan-lake/resources/featured-scenarios/create-your-first-connector-with-ballerina.md

```diff
@@ -0,0 +1,176 @@
+   - redefining inline schemas as named schemas to avoid duplication and improve readability.
+
+4. Update the `sanitations.md` file (under `docs/spec/`) with the details of any changes made to the original OpenAPI specification.
+
+> **Note:** You may need to perform additional sanitization after generating the client code (Step 3) and testing the connector (Step 4) to address any compile-time or runtime issues. Make sure to update the `sanitations.md` file accordingly.
+
+## Step 3: Generate the Ballerina client code
+
+With your OpenAPI spec ready, use the [Ballerina OpenAPI tool](https://ballerina.io/learn/openapi-tool/) to generate the connector code.
+
+1. In your terminal, run the following command from the project root:
+      
+   ```
+   bal openapi -i path/to/spec --mode client -o ballerina
+   ```
+
+   This will generate the ballerina source code of the client connector in your `ballerina/` directory.
+
+> **Note:** The Ballerina OpenAPI tool supports multiple customization options when generating clients. For more details, check the [Ballerina OpenAPI tool documentation](https://ballerina.io/learn/openapi-tool/).
+
+2. Make sure that the generated client implementation consists of the following files:
+   - `client.bal`: Contains the client implementation with all the API operations.
+   - `types.bal`: Contains the data types used in the client.
+   - `utils.bal`: Contains utility functions used in the client.
+
+## Step 4: Test the connector
+
+Now that your connector is generated, it is important to write tests to ensure everything works as expected.
+
+1. In the `ballerina/tests` directory, add the required test files and write test cases for the key operations of your connector. Aim to cover as many API use cases as possible.
+
+2. Execute the tests using the `bal test` command to verify the functionality of your connector.
+
+> For detailed guidance on writing tests, check the [Ballerina testing guide](https://ballerina.io/learn/test-ballerina-code/test-a-simple-function/).
+
+> For sample implementations, refer to the [test implementation of the Ballerina Twitter connector](https://github.com/ballerina-platform/module-ballerinax-twitter/tree/main/ballerina/tests).
+
+## Step 5: Document the connector
+
+Follow these steps to ensure your connector is well-documented:
+
+1. Update `Module.md` and `Package.md` files
+
+   These files will be displayed on the Ballerina Central package landing page. Ensure they introduce and explain the connector package clearly and comprehensively. Each file should include the following sections:
+   
+   - **Overview**: Provide a concise introduction to the connector, explaining its purpose and key features.
+   - **Setup**: Offer step-by-step instructions on configuring the connector and any necessary prerequisites, such as API keys or environment setup.
+   - **Quickstart**: Include a simple, clear example that enable users to start using the connector immediately.
```

Comment:
`enable` -> `enables`

Reply:
Updated in https://github.com/ballerina-platform/ballerina-dev-website/pull/9507.